examples/autonat/src/bin/autonat_client.rs

```diff
@@ -64,7 +64,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     let transport = tcp::async_io::Transport::default()
         .upgrade(Version::V1Lazy)
-        .authenticate(noise::NoiseAuthenticated::xx(&local_key)?)
+        .authenticate(noise::Config::new(&local_key)?)
```

Comment:
Needs a version bump of the `libp2p-noise` dependency in the autonat client example `Cargo.toml`, right?

Reply:
Right! That is accessed via `libp2p` so I bumped it there.